audio/audacity/Portfile

```
@@ -271,8 +272,30 @@ configure.cxxflags-append \
                     -Wno-inconsistent-missing-override \
                     -Wno-overloaded-virtual
 
+platform darwin {
+    # set the install_rpath first
+    cmake.install_rpath-append \
+                    ${prefix}/lib/audacity \
+                    ${aud_app_path}/Contents/Frameworks
+    if {[variant_isset wxsystem]} {
+        cmake.install_rpath-append \
+                    ${wxWidgets.prefix}/lib
+    } else {
+        configure.ldflags-append \
+                    -Wl,-rpath,${prefix}/lib/audacity \
+                    -Wl,-rpath,${aud_app_path}/Contents/Frameworks
+    }
+    if {${subport} eq "${name}"} {
```

Quotes are not needed.
```suggestion
    if {${subport} eq ${name}} {
```